src/main/java/werkIt/UI.java

```diff
@@ -119,4 +119,79 @@ public void printNewWorkoutCreatedMessage(Workout newWorkout) {
         System.out.println("\t" + newWorkout.toString());
         System.out.println();
     }
+
+    /**
+     * Prints the text in colors pre-defined in TextColor class.
+     * @param color ANSI color codes defined in TextColor class ONLY.
+     * @param text The string text that needs to be colored.
+     */
+    public void printColorText(String color, String text) {
+        System.out.println(color + text + COLOR_RESET);
+    }
+
+    /**
+     * Prints the help messages. To be updated.
+     */
+    public void printHelpMessage() {
+        printListHelp();
+        printLine();
+        printWorkoutAddHelp();
+        printLine();
+        printWorkoutDeleteHelp();
+        printLine();
+        printWorkoutUpdateHelp();
+        printLine();
+        printExitHelp();
+    }
+
+    /**
+     * Prints help message for 'list' command.
+     */
+    public void printListHelp() {
+        System.out.println("\t To view all workouts, please enter:");
+        printColorText(COLOR_YELLOW, "\t workout /list");
+        System.out.println("\t This will print all the existing workouts.");
+    }
+
+    /**
+     * Prints help message for 'workout /new' command.
+     */
+    public void printWorkoutAddHelp() {
+        System.out.println("\t To add a workout, please enter: ");
+        printColorText(COLOR_YELLOW, "\t workout /new <exercise name> /reps <no. of repetitions>");
+        System.out.println("\t Example: ");
+        printColorText(COLOR_YELLOW, "\t workout /new push up /reps 10");
+        System.out.println("\t This will add a workout with 10 reps of push up.");
+    }
+
+    /**
+     * Prints help message for 'workout /delete' command.
+     */
+    public void printWorkoutDeleteHelp() {
+        System.out.println("\t To delete a workout, please enter: ");
+        printColorText(COLOR_YELLOW, "\t workout /delete <index>");
+        System.out.println("\t Example: ");
+        printColorText(COLOR_YELLOW, "\t workout /delete 1");
+        System.out.println("\t This will delete the workout with index 1 if exists.");
+    }
+
+    /**
+     * Prints help message for 'workout /update' command.
+     */
+    public void printWorkoutUpdateHelp() {
+        System.out.println("\t To update a workout, please enter: ");
+        printColorText(COLOR_YELLOW, "\t workout /update <index> <quantity>");
+        System.out.println("\t Example: ");
+        printColorText(COLOR_YELLOW, "\t workout /update 1 15");
+        System.out.println("\t This will update the workout with index 1 to 15 reps if exists.");
+    }
+
+    /**
+     * Prints help message for 'exit' command.
+     */
+    public void printExitHelp() {
+        System.out.println("\t To exit werkIt, please enter: ");
+        printColorText(COLOR_YELLOW, "\t exit");
+        System.out.println("\t ☹ This will exit werkIt. :-(");
```

Comment:
I suggest to remove the smiley face at the back and front as the other help commands did not include such emoji. I feel it is a bit out of place.

Reply:
No problems!

Reply:
Issue addressed